src/openrct2/interface/Widget.h

```diff
@@ -47,9 +47,9 @@ enum WINDOW_WIDGET_TYPES
 #define SPINNER_DECREASE(l, r, t, b) (r) - 25, (r)-13, (t) + 1, (b)-1
 #define SPINNER_WIDGETS(colour, left, right, top, bottom, text, tooltip)                                                       \
     { WWT_SPINNER, colour, left, right, top, bottom, text, tooltip },                                                          \
-        { WWT_BUTTON, colour, SPINNER_INCREASE(left, right, top, bottom), STR_NUMERIC_UP, STR_NONE },                          \
+        { WWT_BUTTON, colour, SPINNER_DECREASE(left, right, top, bottom), STR_NUMERIC_DOWN, STR_NONE },                        \
     {                                                                                                                          \
-        WWT_BUTTON, colour, SPINNER_DECREASE(left, right, top, bottom), STR_NUMERIC_DOWN, STR_NONE                             \
+        WWT_BUTTON, colour, SPINNER_INCREASE(left, right, top, bottom), STR_NUMERIC_UP, STR_NONE                               \
```

Comment:
I have changed the wrong spinner widgets: the old ones instead of the new ones. :facepalm: